doc/tech/approach.md

```diff
@@ -0,0 +1,99 @@
+
+> _List needs to be prioritized; expect additional objectives to be added, as they arise. This represents the _What_, not the _How_.
+
+1. The implementation of Networking inside the HW Layer should not be visible to the VNF/CNF and should preferably not even be visible to the IaaS/CaaS
+1. Provide networks for L3 tenant, GWs, SDS, etc.
+1. Cleanly decouple interface/reference points between CNTT constituencies
+1. Provide interoperability at layer demarcation/reference points within the NFVI. Ex.:
+   * any RA couples to RM
+   * like RI couples to RA
+   * like VIs couple to RA
+   * Operators can design or procure a compatible fabric
+1. Concurrently supports containerized and virtualized coexistence for VNF->CNF cutovers, as well as protracted parallel operations
+1. Provide a version controlled catalog of APIs, and their respective spans of control, capabilities and purpose, to facilitate predictable integration w/ a wide selection of fabric implementations
+1. Provide ability for any number of Operator-specific fabrics to power CNTT NFVI
+1. Enable RC's ability to realize mandated OVP qualification deliverables
+1. Unambiguously document the responsibilities of each CNTT constituency
+1. In cases where a VNF/CNF require HW layer resources it should be under the control of the Virtualization Layer
+1. It is important that the HW Infrastructure Manager, each VIM and each VNF/CNF could be managed by separate organizations
+1. Drive the industry towards convergence on ABIs supporting Cloud-Native implementations for SmartNICs
+
+> _**Editor's Note:** Consider moving objectives to a table_
+
+<a name="5.1.3"></a>
+### 5.1.3 Networking Requirements
+
+1. NFVI layer responsibilities will include:
+   * HW Infrastructure Manager shall provide an abstracted model of the allocated HW resources into each specific Virtualization domain
+     * Additionally, it is responsible for maintaining logical isolation between different instances of virtualization domain
+     * Some of the HW resources including networking resources shall be possible to be withheld from Virtualization domains to allow for scaling, spare parts and HW Composition within the HW Infrastructure Layer itself
+   * Virtualization layer shall provide the Cloud Tenants with an abstracted networking environment
+     * It is therefore responsible for maintaining isolation between Cloud Tenants
+
+An example of the layering described above is depicted in **Figure 1-3**, where the Virtualization layer manages the Overlay Networking (e.g., through VLAN allocation) and the HW Infrastructure manages the Underlay networking (e.g., through VxLAN VNI range allocations).
+<p align="center"><img src="./figures/RMCH-01_Network_Layering_v0-3.png" alt="Network Layering" title="Network Layering" width="100%"/></p>
+<p align="center"><b>Figure 1-3:</b> Network Layering</p>
+
+<a name="5.1.4"></a>
+### 5.1.4 Initial Logistical Steps
```

Comment:
Does this section overlap /  can it be merged with #1269?

Reply:
possibly, some cleaning up will happen afterwords.

Reply:
OK, will resolve for this PR and pick it up as a separate exercise.